File: Simulation/Bronze/P3_2016-The_Cow-Signal/cowsignal.py
with open("cowsignal.in", "r") as fin:
    # Read in first line: M = # of lines, N = # of chars in each line, K = multiplier
    M, N, K = map(int, fin.readline().split())
    signal: list[str] = [fin.readline() for row in range(M)]

print("M:", M, "N:", N, "K:", K)
print("Signal:")
for row in signal:
    print(row, end="")
print()

File: Simulation/Bronze/P3_2016-The_Cow-Signal/cowsignal.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
	freopen("cowsignal.in", "r", stdin);
	freopen("cowsignal.out", "w", stdout);
	// First line: m = rows, n = cols, k = multiplier
	int m, n, k;
	cin >> m >> n >> k;
	// For the next m lines, read in n chars and multiply each char by k
	for (int r = 0; r < m; ++r) {
		string currRow;
		for (int c = 0; c < n; ++c) {
			char ch;
			cin >> ch;
			// Multiply each char by k for the current row
			for (int mc = 0; mc < k; ++mc)
				currRow += ch;
		}
		// Multiply the current row by k
		for (int mr = 0; mr < k; ++mr)
			cout << currRow << endl;
	}
	return 0;
}
